lib/pages/settings_pages/settings_page.dart

```diff
@@ -42,7 +42,8 @@ class _SettingsPageState extends State<SettingsPage> {
     Language('de_DE', 'German'),
     Language('el_GR', 'Greek'),
     Language('en_US', 'English'),
-    Language('es_ES', 'Spanish'),
+    Language('es_ES', 'Spanish (Spain)'),
```

Comment:
This should be 'Spanish' only, without Spain

Reply:
yes, I just pushed the commit with the fixed strings!

Reply:
I missed it, it's without "(Spain)" now!